net/sshuttle/Portfile

```diff
@@ -4,25 +4,25 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        sshuttle sshuttle 1.0.1 v
-fetch.type          git
+github.setup        sshuttle sshuttle 1.0.2 v
 revision            0
 
-maintainers         {gmail.com:herby.gillot @herbygillot} openmaintainer
-categories          net
 description         Transparent proxy server that works as a poor man's VPN.
+
 long_description    Transparent proxy server that works as a poor man's VPN. \
                     Forwards over ssh. Doesn't require admin. Works with \
                     Linux and MacOS. Supports DNS tunneling.
 
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+categories          net
 platforms           darwin
 license             LGPL-2
 homepage            https://sshuttle.readthedocs.io/en/stable/
+fetch.type          git
 
-python.default_version          38
-python.versions     27 35 36 37 38
+python.default_version 38
 
 depends_build-append port:py${python.version}-setuptools \
                      port:py${python.version}-setuptools_scm
-
-depends_run-append  port:py${python.version}-setuptools
```

Comment:
you need to change `py-setuptools` to `depends_lib-append` otherwise it will not be available at runtime

Reply:
I think we're good here, `sshuttle` won't need `setuptools` at runtime.

Reply:
no, it does due to its `console_script`:
```
    entry_points={
        'console_scripts': [
            'sshuttle = sshuttle.cmdline:main',
        ],
    },
```

If you look in the file `/opt/local/bin/sshuttle` you will see `from pkg_resources import load_entry_point`, that is part of `py-setuptools` and is therefore needed at runtime as well.

Reply:
OK good point, thanks for catching that - I've updated the `Portfile`.

Reply:
OK, thanks for your patience, @reneeotten  - fixed.

Reply:
okay, that looks good - but now you're also making unrelated white-space changes: these should be in a separate commit from functional ones.